build/helper.py

```diff
@@ -711,7 +717,11 @@ def get_python_type_from_visa_type(visa_type):
     else:
         p_types = importlib.import_module('build.templates.python_types')
     v_type = getattr(p_types, visa_type)
-    p_type = v_type().python_type()
+    # We have a special case for bool
```

Comment:
This is unfortunate...
We have python_types.py so that we can centralize type information... then there is this one special case.
I'm not sure what the best way to solve it is though.